functions/core/updates.sh

```
@@ -7,18 +7,22 @@ check_aether_updates() {
 
     local github_api_url="https://api.github.com/repos/lonersoft/aether/releases/latest"
     
-    echo -e "\e[38;2;129;170;254m[INFO] \e[38;5;250mChecking for updates... this may take a while\e[0m"
+    printout info "Checking for updates... this may take a while"
     
     # Fetch latest release version from GitHub using jq
     local latest_version=$(curl -s --max-time 5 --connect-timeout 5 "$github_api_url" 2>/dev/null | jq -r '.tag_name' 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)